services/Facebook.filters.js

```diff
@@ -1,10 +1,9 @@
 
 export function removeHelpButtons(document) {
-  const links = document.querySelectorAll('a[href="#"][id*="-markdown-internal-id"]');
-  links.forEach(link => {
-    // Helps buttons do not have text content
-    if (!link.text) {
-      link.remove();
+  const imgs = document.querySelectorAll('img[src*="https://scontent"]');
+  imgs.forEach(img => {
+    if(img.parentNode.tagName == 'A' && !img.parentNode.text){
```

Comment:
You may use an intermediate variable to avoir repeating `img.parentNode` like this:
```js
    const parent = img.parentNode;
    if (parent.tagName == 'A' && !parent.text) {
      parent.remove();
    }
```

Reply:
Also, it misses some spaces before and after parenthesis :).

Reply:
Done thanks !